sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchAsyncClient.java

```diff
@@ -1077,19 +1078,24 @@ static SearchRequest createSearchRequest(String searchText, SearchOptions option
     static String createSearchRequestAnswers(SearchOptions searchOptions) {
         QueryAnswerType answer = searchOptions.getAnswers();
         Integer answersCount = searchOptions.getAnswersCount();
+        Double answerThreshold = searchOptions.getAnswerThreshold();
 
         // No answer has been defined.
         if (answer == null) {
             return null;
         }
 
+        String answerString = answer.toString();
+
         // No count, just send the QueryAnswer.
-        if (answersCount == null) {
-            return answer.toString();
+        if (answersCount != null) {
```

Comment:
Unfortunately, I told you the wrong format. If either are set there will need to be `extractive|<option>` but if both are set the format is `extractive|<option 1>,<option 2>`, not `extractive|<option 1>|<option 2>`.